src/PresentationalComponents/Dashboard/Dashboard.js

```diff
@@ -40,69 +40,71 @@ const Dashboard = (/*{ workloads }*/) => {
         chrome.updateDocumentTitle(`Dashboard | Red Hat Insights`);
     }, [chrome]);
 
-    return permission.hasSystems  ?
-        <React.Fragment>
-            <PageSection isWidthLimited variant={ PageSectionVariants.light } className="insd-c-dashboard-header">
-                <Title headingLevel="h1" size="2xl" className="pf-v5-u-screen-reader">
-                    {intl.formatMessage(messages.dashboardTitle)}
-                </Title>
-                <Suspense fallback={ <Loading /> }>
-                    <SystemInventoryHeader />
-                </Suspense>
-            </PageSection>
-            <PageSection isFilled={true} isWidthLimited>
-                <Grid hasGutter>
+    return permission === 'undefined' || permission === null ?
```

Comment:
`permission` never equals to `undefined` or null, and this condition will be never met. See src/App.js

Reply:
@adonispuente, before it is loaded, the `Dashboard` is not rendered: https://github.com/RedHatInsights/insights-dashboard/blob/master/src/App.js#L110, only an empty state. You can try to `console.log(permission)` somewhere above and see that it never gets null or undefined.

Reply:
Also, if we keep this, don't use quotes for undefined, it's a reserved JS type and not a string

Reply:
seems youre right, I could have sworn I got some undefined's. Im getting old, ill send this up